.ci/pitest.sh

```diff
@@ -38,13 +38,20 @@ pitest-annotation|pitest-design \
 |pitest-packagenamesloader \
 |pitest-common-2|pitest-misc|pitest-xpath \
 |pitest-filters \
-|pitest-main \
-|pitest-coding)
+|pitest-main)
   mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
   declare -a ignoredItems=();
   checkPitestReport "${ignoredItems[@]}"
   ;;
 
+pitest-coding)
+  mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
+  declare -a ignoredItems=(
+  "RequireThisCheck.java.html:<td class='covered'><pre><span  class='survived'>                    register(ast, frame);</span></pre></td></tr>"
```

Comment:
Why can't we kill this surviving mutation?

Reply:
Hi @rnveach,
It seems not to be possible to both solve this pitest issue and make `IDEA Inspections Pull Request (Checkstyle)` happy.

This matter has been discussed in 
https://github.com/checkstyle/checkstyle/pull/8013#discussion_r403861806 and https://github.com/checkstyle/checkstyle/pull/8013#discussion_r405303262

Please can you take a look at those? Of course, any suggestion will be welcome.

Reply:
Post pitest report and show what is surviving. I am not understanding how pitest and IDEA are connected to each other.

Reply:
Here is more details
There is only one surviving line which is https://github.com/checkstyle/checkstyle/blob/db0ae506a6bf906e29cb1ea71ecdf27903b1299c/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheck.java#L518

As discussed in https://github.com/checkstyle/checkstyle/pull/8013#discussion_r403861806,  to kill the mutation we need to change the return type from void to `AbstractFrame`.

So we need to change https://github.com/checkstyle/checkstyle/blob/db0ae506a6bf906e29cb1ea71ecdf27903b1299c/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/AbstractFrameCheck.java#L153 to 

```java
protected AbstractFrame register(DetailAST ast, AbstractFrame frame) {
        return frames.put(ast, frame);
}
```

This indeed, solves the pitest issue, but as we've seen in https://github.com/checkstyle/checkstyle/pull/8013#discussion_r405303262, this causes another CI issue from `IDEA Inspections Pull Request (Checkstyle)`
![image](https://user-images.githubusercontent.com/24209192/79077731-d9c28800-7d03-11ea-9d66-449d178b3c70.png)

Reply:
All pitest issues are resolved.

Reply:
CI says pitest is failing.